common/app/com/commercetools/sunrise/sessions/customer/DefaultCustomerInSession.java

```diff
@@ -23,10 +23,12 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(CustomerInSession.class);
     private static final String DEFAULT_CUSTOMER_ID_SESSION_KEY = "sunrise-customer-id";
     private static final String DEFAULT_CUSTOMER_EMAIL_SESSION_KEY = "sunrise-customer-email";
+    private static final String DEFAULT_CUSTOMER_GROUP_SESSION_KEY = "sunrise-customer-group";
```

Comment:
It would be more consistent to name this constant `DEFAULT_CUSTOMER_GROUP_ID_SESSION_KEY` and to use `sunrise-customer-group-id` as key. WDYT?